src/rootPages/Designer/ui_work_object_workspace_popupNewDataField.js

```diff
@@ -415,6 +415,25 @@ export default function (AB, ibase) {
                   // get a new instance of a field:
                   field = this.CurrentObject.fieldNew(vals);
 
+                  if (
+                     vals.columnName !== field.columnName &&
+                     field.columnName === ""
+                  ) {
+                     this.AB.notify.developer(
```

Comment:
I'm not sure that developers need to know this. Could we find a way to show the user (builder) the problem and ask them to rename it?